Octokit/Clients/RepositoriesClient.cs

```diff
@@ -717,6 +725,28 @@ public Task<IReadOnlyList<RepositoryContributor>> GetAllContributors(long reposi
             return ApiConnection.GetAll<RepositoryContributor>(ApiUrls.RepositoryContributors(repositoryId), parameters, options);
         }
 
+        /// <summary>
+        /// Gets all topics for the specified repository.
+        /// </summary>
+        /// <remarks>
+        /// See the <a href="https://docs.github.com/en/rest/reference/repos#get-all-repository-topics">API documentation</a> for more details
+        /// </remarks>
+        /// <param name="owner">The owner of the repository</param>
+        /// <param name="name">The name of the repository</param>
+        /// <returns>All topics associated with the repository.</returns>
+        [ManualRoute("GET", "/repos/{owner}/{repo}/topics")]
+        public async Task<IReadOnlyList<string>> GetAllTopics(string owner, string name)
```

Comment:
❓ Are we OK with a list of `string` for now, since that's definitely what they are? Or do we require the idea of a strongly-typed `RepositoryTopic` object? Given that there's only one field, I'm inclined to think that using the primitive is OK, but happy to adjust if others disagree.

Reply:
Yep, this is fine